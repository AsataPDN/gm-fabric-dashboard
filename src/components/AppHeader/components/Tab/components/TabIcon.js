import { spacingScale } from "../../../../../style/styleFunctions";
import styled from "styled-components";

const TabIcon = styled.span`
  width: ${spacingScale(4)};
  display: flex;
  align-items: center;
`;

export default TabIcon;
