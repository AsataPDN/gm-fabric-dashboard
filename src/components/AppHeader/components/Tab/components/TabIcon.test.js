import React from "react";
import { shallow } from "enzyme";
import TabIcon from "./TabIcon";

describe("TabIcon", () => {
  it("matches snapshot", () => {
    const aTabIcon = shallow(<TabIcon />);
    expect(aTabIcon).toMatchSnapshot();
  });
});
