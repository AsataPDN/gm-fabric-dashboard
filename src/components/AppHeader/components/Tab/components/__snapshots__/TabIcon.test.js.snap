// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabIcon matches snapshot 1`] = `
<styled.span>
  <Icon />
</styled.span>
`;
